docs/modules/installation/pages/installation.adoc

```diff
@@ -0,0 +1,38 @@
+= Installation
+:imagesdir: ../assets/images
+
+## Deployment via KAR-file
+
+The installation of the plugin is simple.
+Just copy the provided KAR-file to your *OpenNMS* `deploy` directory.
+The following example copies the KAR-file `path/to/opennms-velocloud-plugin.kar` to the `deploy` directory assuming that *OpenNMS* is installed in `/opt/opennms`.
+
+```
+$ cp path/to/opennms-velocloud-plugin.kar /opt/opennms/deploy/
+```
+
+After this step you are able to install the plugin in the *OpenNMS* Karaf shell:
+
+```
+admin@opennms()> feature:install opennms-plugins-velocloud
```

Comment:
Is this step really required? As far as I understand copying the file is enough.

Reply:
As far as I know you need this step. In order to start it immediately you need an entry in `$OPENNMS_HOME/etc/featuresBoot.d`. The Alec documentation looks similar.

Reply:
just moving the file into the deploy folder won't install it, the karaf command is needed to install.